web/src/App.jsx

```diff
@@ -18,6 +19,7 @@ function App() {
 				<Route path="/subscribe" element={<Subscribe />} />
 				<Route path="/subscribe/confirmation" element={<ConfirmationPage />} />
 				<Route path="/subscribe/error" element={<ErrorOnSubscribe />} />
+				<Route path="/upload/FileUploading" element={<FileUploading />} />
```

Comment:
```suggestion
				<Route path="/upload" element={<FileUploading />} />
```

You don't need to include the component name in the path. This `path` value is what appears in the browser url bar

Reply:
Done